playbooks/system-criticality.md

```diff
@@ -31,21 +31,24 @@ Retired systems, spikes, or time-bound experiments that aren't significantly use
-| Support                      | Fixes are prioritized with respective teams' backlogs rather than #incidents | Incident response time &lt;30 min. during waking hours                                                     | Incident response time &lt;5 min.                                                                                                                                                 |
-| Data                         |                                                                              | Weekly backups                                                                                             | Daily backups                                                                                                                                                                     |
-| Security                     |                                                                              | Known vulnerabilities patched within 30 days                                                               | Known vulnerabilities patched within 7 days                                                                                                                                       |
+## System expectations (draft)
+
+Each level has corresponding expectations for how the system is built, operated, and maintained. These are _target_
+expectations and existing systems may not fully comply [yet]. New systems should aim to abide by these as much as
+possible.
+
+|                   | Level 1                                                                                                                                                                                                    | Level 2                                                  | Level 3                                                                                                                                 |
+| ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| Development       | READMEs contain up-to-date set-up instructions<br/>Code review on all [pull requests](/playbooks/engineer-workflow.md#pull-requests)<br/>New systems or significant new components undergo [tech review]() | &larr;ditto                                              | &larr;ditto +<br/>Production environment is replicable locally (TODO)                                                                   |
```

Comment:
@eessex you mentioned this concern about having high-fidelity local (or staging) environments available. Is there better language we could use here, or an example?

Reply:
Also, it would be nice to have a link that elaborates on what "technical review" means here. A technical plan format is proposed in https://github.com/artsy/README/issues/245, and that overlaps with a technical design review (e.g., it can serve as a starting point for a review discussion).